src/Nox.Generator/Presentation/Api/OData/EntityControllerGenerator.cs

```diff
@@ -214,7 +214,8 @@ private static void GenerateDelete(Entity entity, string entityName, CodeBuilder
     private static void GeneratePut(Entity entity, CodeBuilder code, NoxSolution solution)
     {
         // Method Put
-        code.AppendLine($"public async Task<ActionResult> Put({PrimaryKeysFromRoute(entity, solution)}, [FromBody] {entity.Name}UpdateDto {entity.Name.ToLowerFirstChar()})");
+        code.AppendLine($"[EnableQuery]");
```

Comment:
I think for now we do not want to enable this on the single entity returned by put